observations/obs_converters/even_sphere/README.rst

```diff
@@ -2,75 +2,330 @@
+   T_error_var = [1.44 0.64 0.64 0.81 1.44 0.64];
+   W_error_var = [1.96 2.25 4.41 9.00 7.29 4.41];
+   even_sphere(nprofiles, 'levels', levels, ...
+              'T_error_var', T_error_var, 'W_error_var', W_error_var)
+
+
+2. 30 horizontal locations at 3 pressure levels. Note that the
+   *nlevels* argument specifies that only the first 3 pressure levels
+   are used even though there are 6 potential pressure levels. 
+   Similarly, only the matching error variances are used.
+
+.. code-block::
+
+   nprofiles   = 30;
+   nlevels     = 3 ;
+   levels      = [1000  850  500  300  200  100];
+   T_error_var = [1.44 0.64 0.64 0.81 1.44 0.64];
+   W_error_var = [1.96 2.25 4.41 9.00 7.29 4.41];
+   even_sphere(nprofiles, 'nlevels', nlevels, 'levels', levels, ...
+              'T_error_var', T_error_var, 'W_error_var', W_error_var)
+
+Levels
+------
+ 
+.. attention::
+
+   Specifying the vertical levels is the most problematic part of this process.
+   Since observation error variances tend to vary with level, coordinating
+   the levels and the error variances is required.
+
+The *mandatory pressure levels* defined in the
+`AMS glossary <https://glossary.ametsoc.org/wiki/Mandatory_level>`_ and their corresponding error variances are
+
+.. code::
+
+   levels      = [1000  925  850  700  500  400  300   250  200  150  100   70   50   30   20   10    7    5    3    2    1];
+   T_error_var = [1.44 1.00 0.64 0.64 0.64 0.64 0.81  1.44 1.44 1.00 0.64 0.64 0.81 1.00 1.69 2.25 2.25 2.25 2.25 2.25 2.25];
+   W_error_var = [1.96 2.25 2.25 2.56 4.41 6.76 9.00 10.24 7.29 5.76 4.41 4.41 4.41 4.41 4.41 4.41 4.41 4.41 4.41 4.41 4.41];
+
+The default levels that this program generates were use to create the 
+Zagar OSSE which did not assimilate any observations above (about) 36 hPa.
+The default pressure levels and corresponding observation error variances are: 
+
+.. code::
+
+   levels      = [1000  925  850  700  500  400  300   250  200  150  100   70   50   40];
+   T_error_var = [1.44 1.00 0.64 0.64 0.64 0.64 0.81  1.44 1.44 1.00 0.64 0.64 0.81 1.00];
+   W_error_var = [1.96 2.25 2.25 2.56 4.41 6.76 9.00 10.24 7.29 5.76 4.41 4.41 4.41 4.41];
+
+The following levels may be useful since we rarely assimilate observations above 150 hPa
+and there are extra levels near the surface because any topography will interfere
+with the 1000 hPa level. These extra levels are **not** *mandatory* levels.
+
+.. code::
+
+   levels      = [1000  950  900  850  800  750  700  650  600  550  500  400  300  200  150];
+   T_error_var = [1.44 1.21 0.81 0.64 0.64 0.64 0.64 0.64 0.64 0.64 0.64 0.64 0.81 1.44 1.00];
+   W_error_var = [1.96 2.25 2.25 2.25 2.56 2.56 2.56 3.24 3.61 4.00 4.41 6.76 9.00 7.29 5.76];
+   % OR - to replicate what the (deprecated) even_sphere_dense.m  did:
+   T_error_var = [1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00];
+   W_error_var = [4.00 4.00 4.00 4.00 4.00 4.00 4.00 4.00 4.00 4.00 4.00 4.00 4.00 4.00 4.00];
```

Comment:
This example replicates the output from `even_sphere_dense.m`, so I removed `even_sphere_dense.m` from the repo.

Reply:
i'm not sure this makes sense.  even_sphere_dense had hardcoded a larger number of obs to generate, thus the 'dense'.  those error values are what was in both even_sphere and even_sphere_dense originally.  kevin replaced them with more realistic errors.  this is just confusing now.  i'd delete lines 168 to 170.  if someone wants to make output with more obs in it, they update the namelist now.

Reply:
I agree that even the mentions of even_sphere_dense are unnecessary.
Just a note, that users are free to define levels and variances to suit their needs, should be enough.

I think the Attention message is more alarming than it needs to be. How about 
"If you need realistic error variances attached to your observations,
be careful to align your levels and variances."

The ability to specify nprofiles < size(levels) makes other documentation simplifications possible.
I suggest (at some point) making the AMS glossary mandatory levels be the default.
People can use those up to whatever height they want, just by specifying nlevels, as already described.
Then use the "extra low levels and truncated top"  as an example of nonmandatory levels 
(and why people might want them) and simplified variances.  
I think the Zagar example doesn't add much, except historically.

Reply:
your suggested changes sound good.  this is your pull request, so you can update this file with the first two suggestions.  the other comments about profiles seems like enough of a change to wait for another pull request and maybe goes into an issue at some point.

Reply:
I couldn't resist changing the default levels and the description.